src/fw/applib/ui/kino/kino_player.c

```diff
@@ -212,7 +211,14 @@ void kino_player_draw(KinoPlayer *player, GContext *ctx, GPoint offset) {
   }
 }
 
+void kino_player_draw_processed(KinoPlayer *player, GContext *ctx, GPoint offset,
```

Comment:
change `kino_player_draw()` to call this function to have a clear hierarchy and avoid divergence in behavior? See `kino_reel_draw()` and `kino_reel_draw_processed()` for reference.